app/src/lib/pocket.ts

```diff
@@ -65,6 +66,7 @@ export async function getAppsInNetwork(): Promise<Omit<Application, 'toJSON' | '
   const rpcResponse = await pocketInstance.rpc(rpcProvider)?.query.getApps(undefined, BigInt(0), undefined, page, perPage)
 
   if (typeGuard(rpcResponse, RpcError)) {
+    log('error', 'failed retrieving applications from network', rpcResponse.message)
```

Comment:
Hmm is this right? This is calling `log` from './logger` but in other files, this is being used as an object that contains a `log()` function.